Source/Engine/Core/Math/Quaternion.cs

```diff
@@ -999,7 +999,7 @@ public static void RotationLookAt(ref Vector3 forward, ref Vector3 up, out Quate
         /// <param name="forward">The camera's forward direction.</param>
         /// <param name="up">The camera's up vector.</param>
         /// <returns>The created look-at quaternion.</returns>
-        public static Quaternion RotationLookAt(Vector3 forward, Vector3 up)
+        public static Quaternion RotationLookAt(Vector3 forward, Vector3 up = Vector3.Up)
```

Comment:
The default value must be a compile time constant, that's why it's not possible to set a custom value, C# limitation.
There is a workaround using nullable value but it's not very clean imo.

Reply:
fixed